src/connectedk8s/azext_connectedk8s/_utils.py

```diff
@@ -567,6 +567,8 @@ def helm_install_release(chart_path, subscription_id, kubernetes_distro, kuberne
         cmd_helm_install.extend(["--set", "global.isProxyEnabled={}".format(True)])
     if container_log_path is not None:
         cmd_helm_install.extend(["--set", "systemDefaultValues.fluent-bit.containerLogPath={}".format(container_log_path)])
+    if enable_oidc_issuer:
```

Comment:
if it is false,, can you explicitly set to false for the update to false scenario